### ISSUE TITLE
Deceision tree

### ISSUE THREAD

**Issue by gtech04** [gtech04/ML-LAB-S5#8]
Algorithm:
Step 1:Import Libraries:
Step 2:Load and Explore Dataset:
step 3:Read the dataset from 'car_evaluation.csv' using pd.read_csv.
Step 4:Set column names using colnames.
Step 5:Display a count plot for the 'decision' variable.
Step 6:Preprocess Data
Step 7:Use LabelEncoder to encode categorical variables in the dataset.
Step 8:Display the value counts for the 'decision' column.
Step 9:Split Dataset
Step 10:Build and Train Decision Tree Model.
Step 11:Fit the model using the training data.
Step 12:Predict the 'decision' variable for the test set.
Step 13:Evaluate Model:Compute the confusion matrix using confusion_matrix.
Step 14:Display the confusion matrix using ConfusionMatrixDisplay.
